### ISSUE TITLE
How to set a direct parent div's height, to height of scrollbar?

### ISSUE THREAD

**Issue by SaadBazaz** [rintoj/ngx-virtual-scroller#465]
Hello!

So the issue I face is a little confusing.

I used the basic example in the repo to create a list of 10,000 numbers and assigned parentScroll to the window, which is successfully being virtually scrolled.

Then I have nested the list into a card (having a card in the back of the list is a design necessity):

```
<mat-card class="custom-card">
    <virtual-scroller #scroll [items]="myItems" [parentScroll]="scroll.window">
      <div *ngFor="let item of scroll.viewPortItems">{{item }}</div>
    </virtual-scroller>
</mat-card>
```

But even with height: 100% and flex:1, it shows the card like this:
![image](https://user-images.githubusercontent.com/51885228/89194440-ca23c380-d5c0-11ea-9e18-12df9edca3e5.png)

As highlighted in the console, document.documentElement.scrollHeight only shows 696, which is in reality the clientHeight.

In other words, is there any way to expand this card's height to fit all the content?
Is there any variable from ngx-virtual-scroller I could use? And how?

**Comment by vmalvaro** [comment]
Can you create a [Stackblitz](https://stackblitz.com/), with an example?, if you set up a fixed height it works?

**Comment by SaadBazaz** [comment]
Hey @vmalvaro , I've built a repro
https://angular-nxjd37.stackblitz.io

**Comment by SaadBazaz** [comment]
I'd really appreciate for someone to drop a follow-up on this!

**Comment by kmcs** [comment]
Work around works for me.
Idea to get it work not with fixed pixel you have to set a height on each parent: https://stackoverflow.com/questions/31728022/why-is-percentage-height-not-working-on-my-div/31728799#31728799
But it's even harder as your workaround... it would be nice if the virtual-scroll-element has a property so own calculation is not needed.

**Comment by SaadBazaz** [comment]
Is this a solution to our problem? -> https://github.com/rintoj/ngx-virtual-scroller/pull/447

I'd like to know if anyone has tried this out to solve our particular issue.
